packages/react-baseline-inputs/src/RenderRequiredDefault.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from "react";
+import { RenderRequired } from "./types";
+
+export const RenderRequiredDefault: RenderRequired = props => {
```

Comment:
This should probably just be a function, not a component. Thus, the `R` should be lower case.